### PR TITLE
Fix package name

### PR DESCRIPTION
**Pull request by pklimai**
There is a typo in package name in kmath-for-real tests.

### CODE REVIEW

**Comment by CommanderTvis** [comment]
@pklimai It should be even `space.kscience`.

File: kmath-for-real/src/commonTest/kotlin/kscience/kmath/real/DoubleMatrixTest.kt [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
  */
 
-package kaceince.kmath.real
+package kscience.kmath.real
```
Comment (CommanderTvis) [inline kmath-for-real/src/commonTest/kotlin/kscience/kmath/real/DoubleMatrixTest.kt:6]:
```suggestion
package space.kscience.kmath.real
```

File: kmath-for-real/src/commonTest/kotlin/kscience/kmath/real/DoubleVectorTest.kt [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
  */
 
-package kaceince.kmath.real
+package kscience.kmath.real
```
Comment (CommanderTvis) [inline kmath-for-real/src/commonTest/kotlin/kscience/kmath/real/DoubleVectorTest.kt:6]:
```suggestion
package space.kscience.kmath.real
```

File: kmath-for-real/src/commonTest/kotlin/kscience/kmath/real/GridTest.kt [hunk 3]
```diff
@@ -3,7 +3,7 @@
  * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
  */
 
-package kaceince.kmath.real
+package kscience.kmath.real
```
Comment (CommanderTvis) [inline kmath-for-real/src/commonTest/kotlin/kscience/kmath/real/GridTest.kt:6]:
```suggestion
package space.kscience.kmath.real
```

**Review by CommanderTvis — CHANGES_REQUESTED** [review]

**Comment by pklimai** [comment]
@CommanderTvis fixed

**Review by altavir — APPROVED** [review]

**Review by CommanderTvis — APPROVED** [review]

**Pull request merged**